scripts/ci/grid.sh

```diff
@@ -5,17 +5,25 @@ set -e
 SCRIPT_DIR=$(dirname ${0})
 
 setupNvm() {
-    export NVM_DIR="$HOME/.nvm"
-    [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"  # This loads nvm
-
-    nvm install
-    nvm use
+  export NVM_DIR="$HOME/.nvm"
+  if [ -s "$NVM_DIR/nvm.sh" ]; then
+    # This loads nvm if installed directly
+    source "$NVM_DIR/nvm.sh"
+  elif [ -s "$(brew --prefix nvm)/nvm.sh" ]; then
+    # This will load nvm if installed via brew
+    source "$(brew --prefix nvm)/nvm.sh"
```

Comment:
As this is the CI step, would we ever expect nvm to be installed via brew here?

Reply:
Fair point, but I was testing the CI step locally so this was useful for me and I'd anticipate someone doing the same in the future.